lib/config/rules/typescript.js

```diff
@@ -11,18 +11,23 @@ module.exports = {
   '@typescript-eslint/interface-name-prefix': 'off',
```

Comment:
I noticed in one of the projects that I was getting a lint error for both the TypeScript `camelcase` rule and the Babel one. Is that resolved by this PR?

Reply:
I remember noticing that at one point as well. It is solved by this PR as shown in the quilt companion PR here: https://github.com/Shopify/quilt/pull/1110/files#diff-d8cfb868aeea19ce239de10dcfdbd4c2R55